include/seqan3/alphabet/nucleotide/sam_dna16.hpp

```diff
@@ -0,0 +1,191 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2019, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2019, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE.md
+// -----------------------------------------------------------------------------------------------------
+
+/*!\file
+ * \author Svenja Mehringer <svenja.mehringer AT fu-berlin.de>
+ * \brief Contains seqan3::sam_dna16.
+ */
+
+#pragma once
+
+#include <seqan3/alphabet/nucleotide/nucleotide_base.hpp>
+
+namespace seqan3
+{
+
+/*!\brief A 16 letter DNA alphabet, containing all IUPAC symbols minus the gap and plus an equality sign ('=').
+ * \ingroup nucleotide
+ * \implements seqan3::WritableAlphabet
+ * \implements seqan3::NucleotideAlphabet
+ * \implements seqan3::TriviallyCopyable
+ * \implements seqan3::StandardLayout
+ * \if DEV \implements seqan3::detail::ConstexprAlphabet \endif
+ *
+ * \details
+ *
+ * The seqan3::sam_dna16 alphabet is used for aligned sam_dna16 sequences with the extra alphabet character
+ * represented by '=', which denotes that the aligned base is equal to the respective reference sequence.
```

Comment:
The seqan3::sam_dna16 alphabet is the nucleotide alphabet used inside the SAM, BAM and CRAM formats.
It has all the letters of the seqan3::dna15 alphabet and the extra alphabet character '=' which denotes a nucleotide character *identical to the reference*. Without the context of this reference sequence, no assumptions can be made about the actual value of `=` letter.